### PR TITLE
Server 1.6.0

### PR DESCRIPTION
**Pull request by nbeguier**
- Display ssh key fingerprint and rate it.
- Improve get_fingerprint function

and... some fix
- reorder import
- remove unnecessary else
- rename the boolean list (built-in function) to is_list

### CODE REVIEW

File: src/server/ssh_utils/__init__.py [hunk 1]
```diff
@@ -6,13 +6,13 @@
 
 def get_fingerprint(public_key_filename):
     """
-    Return Fingerprint
+    Returns a key fingerprint
     """
     try:
-        fingerprint = check_output([
+        fingerprint = '\n'.join(check_output([
             'ssh-keygen',
```
Comment (xakraz) [inline src/server/ssh_utils/__init__.py:13]: FAOR: Is there no OpenSSH pure python lib rather than calling system `ssh-keygen` ?

**Review by xakraz — APPROVED** [review]
LGTM

**Comment by xakraz** [comment]
It seem that Travis build has not been triggered ...
I am closing and re-opening the PR ;)

**Comment by xakraz** [comment]
By the way if client and server are not always in sync, we have to re-define the way we tag the docker images ^^

File: src/server/ssh_utils/__init__.py (hunk 1 above)
Comment (vdemonchy) [reply src/server/ssh_utils/__init__.py:13]: https://www.paramiko.org/ might be the answer :)

**Comment by nbeguier** [comment]
Does someone would like to review this PR ? I have some more fix to push !

**Comment by xakraz** [comment]
FOAR: Since it is the public Github.com repos, it might be a "good" habit to wait more than 1 hour when asking a question and doing the merge ;)

**Comment by nbeguier** [comment]
In fact, I have merged it because you have already review this PR and accept it.
`xakraz approved these changes 26 days ago`

So, I asked my question before checking the validation, and I think you made the same mistake right now :p

**Comment by xakraz** [comment]
Ok no pb, I was just suggesting, nothing personal.
And about my mistake, are you referring the "delete branch" ?

**Pull request merged**